metadata/metadata_service/proxy/gremlin_proxy.py

```diff
@@ -1958,8 +1958,8 @@ def _get_edge_type_from_user_resource_rel_type(self, relation: UserResourceRel)
 
         raise NotImplementedError(f"Don't know how to handle UserResourceRel={relation}")
 
-    def _parse_lineage(self, resource_type: ResourceType, type_: str, upstream_tables: List[LineageItem], path: Path,
-                       downstream_tables: List[LineageItem]) -> Tuple[List[LineageItem], List[LineageItem]]:
+    def _parse_lineage(self, resource_type: ResourceType, type_: str, upstream_tables: List[LineageItem],
```

Comment:
what is the main reason of the argument swap?

Reply:
Previously, `parse_lineage` function accepted arguments in the wrong order as it was called on line 2038. There are two possible fixes for this:

1) Align the order of the arguments in `parse_lineage` and line 2038.
2) Add kwargs 

Technically, either one of the above fixes would work but I have included both.